src/Umbraco.Web.UI.Client/src/init.js

```diff
@@ -18,23 +18,26 @@ app.run(['userService', '$log', '$rootScope', '$location', 'queryStrings', 'navi
         eventsService.on("app.authenticated", function(evt, data) {
             
             assetsService._loadInitAssets().then(function() {
-                
-                //Register all of the tours on the server
-                tourService.registerAllTours().then(function () {
-                    appReady(data);
-                    
-                    // Auto start intro tour
-                    tourService.getTourByAlias("umbIntroIntroduction").then(function (introTour) {
-                        // start intro tour if it hasn't been completed or disabled
-                        if (introTour && introTour.disabled !== true && introTour.completed !== true) {
-                            tourService.startTour(introTour);
-                        }
-                    });
 
-                }, function(){
-                    appReady(data);
+                // Loads the user's locale settings for Moment.
+                userService.loadMomentLocaleForCurrentUser().then(function() {
```

Comment:
This is kind of odd with the sub promise chaining, you can instead use $q.all and give it a list of promises, here's a good example of exactly this topic https://www.martin-brennan.com/using-q-all-to-resolve-multiple-promises/